grasp_detection_position: [ [0.0, 0.0, 0.0, 0.0, -1.57, 0.0],
                            [0.5, 0.185, -0.264, -0.4, -1.4, 0.0],
                            [-0.5, 0.185, -0.264, 0.4, -1.4, 0.0]
                          ]
place_position: [-1.708, -0.330, -0.355, -0.054, -0.811, 0.005]
resting_position: [0.0, 0.640187, -1.397485, 0.0, 0.0, 0.0]
